Integrations/DigitalDefense/DigitalDefense.py

```diff
@@ -0,0 +1,596 @@
+    vulns = get_vulns(severity, min_severity, days_older_than, days_newer_than, host_id)
+    return get_vuln_outputs(vulns)
+
+
+#########################
+# CREATING SCAN METHODS:
+#########################
+class _tz_UTC(tzinfo):
+    ''' UTC '''
+    def utcoffset(self, dt):
+        return timedelta(0)
+
+    def tzname(self, dt):
+        return 'UTC'
+
+    def dst(self, dt):
+        return timedelta(0)
+
+
+def ip2long(ip_address):
+    ''' Converts IP from string to number. '''
+    return struct.unpack("!L", socket.inet_aton(ip_address))[0]
+
+
+def long2ip(ip_address):
+    ''' Converts IP from number to string. '''
+    return socket.inet_ntoa(struct.pack("!L", ip_address))
+
+
+def get_network_data():
+    try:
+        url = FVM_URL + "/api/networkprofiles/?_0_eq_networkprofile_internal=True"
+        resp = requests.get(url, headers={'Authorization': 'Token ' + str(API_TOKEN)})
+        resp.raise_for_status()
+        return json.loads(resp.text)
+    except Exception as err:
+        print('ERROR getting network data: ' + str(err))
+    return None
+
+
+def get_scan_data(network_data, low_ip, high_ip):
+    for profile in network_data:
+        #If there is no scanner using this profile, then continue to the next profile
+        if len(profile.get('scanner_names', "")) == 0:
+            continue
+        status_url = FVM_URL + "/api/scanners/" + str(profile.get('scanner_ids')[0]) + "/"
+        status_resp = requests.get(status_url, headers={'Authorization': 'Token ' + str(API_TOKEN)})
+        status_resp.raise_for_status()
+        status = json.loads(status_resp.text)
+        if status.get('status', '') == 'online':
+            url = FVM_URL + "/api/networkprofiles/" + str(profile['id']) + "/rules/"
+            profile_data = []
+            current_data = None
+            have_all_data = False
+            while not have_all_data:
+                resp = requests.get(url, headers={'Authorization': 'Token ' + str(API_TOKEN)})
+                resp.raise_for_status()
+                current_data = json.loads(resp.text)
+                profile_data.extend(current_data.get('results', []))
+                if (current_data.get('next', None)):
+                    url = current_data.get('next')
+                else:
+                    have_all_data = True
+            for rule in profile_data:
+                if rule.get('ip_address_range', None):
+                    if (rule['ip_address_range']['high_ip_number'] >= high_ip) and (rule['ip_address_range']['low_ip_number'] <= low_ip):
+                        return json.dumps({
+                            'scanner_name': profile['scanner_names'][0],
+                            'scanner_id': profile['scanner_ids'][0],
+                            'profile_name': profile['name'],
+                            'profile_id': profile['id']
+                        })
+    return_error("Error: no scan data found.")
```

Comment:
Scan data is critical to performing a scan. This is because if no scanner is available, we are unable to perform a scan and unable to continue the command `frontline-scan-asset`